llvmDebugInfoC/src/scripts/konan_lldb.py

```diff
@@ -231,7 +244,15 @@ def offset(self):
 
 class KonanObjectSyntheticProvider(KonanHelperProvider):
     def __init__(self, valobj, tip):
-        super(KonanObjectSyntheticProvider, self).__init__(valobj)
+        # Save an extra call into the process
+        if tip in SYNTHETIC_OBJECT_LAYOUT_CACHE:
```

Comment:
if tip in SYNTHETIC_OBJECT_LAYOUT_CACHE, we don't need to call 'Konan_DebugGetFieldCount'